bcda/cclf/fileprocessor_test.go

```diff
@@ -217,6 +222,19 @@ func TestGetCCLFMetadata(t *testing.T) {
 				acoID:     sspID,
 				timestamp: validTime,
 				perfYear:  20,
+				fileType:  models.FileTypeDefault,
+			},
+		},
+		{
+			"Runout SSP file", sspID, sspRunoutFile, "",
```

Comment:
Does it make sense to test runout files for the other ACO types?

Reply:
I was going back and forth on this. Since we've only guaranteed that we'll receive the runout files for SSP ACOs, I decided to leave them off.

Once we have the conversations with CEC/NGACO (and other types), we  can add those tests in. Just in case those have a different format.